hack/.structured_logging

```diff
@@ -1,2 +1,3 @@
 cmd/kubelet/...
+pkg/controller/certificates/...
```

Comment:
Please don't add new directories here, we are not yet committing to migrating other directories.

Reply:
Removed.